app/models/ability.rb

```diff
@@ -0,0 +1,17 @@
+class Ability
+  include CanCan::Ability
```

Comment:
as I recall you worked with cancancan, but wasn't familiar with Pundit?
if you did, please try integration of Pundit instead
this project is about playing around and not testing your skills anyway, so using something you are not familiar with will be much more useful